_ko/tour/annotations.md

```diff
@@ -19,21 +19,6 @@ previous-page: automatic-closures
 
 어노테이션 절은 첫 번째 정의나, 그 다음에 이어지는 선언에 적용된다. 정의와 선언에는 하나 이상의 어노테이션 절이 붙을 수 있다. 이런 절이 표현되는 순서는 영향을 미치지 않는다.
 
-어노테이션 절의 의미는 _구현 종속적_ 이다. 자바 플랫폼에선 다음의 스칼라 어노테이션이 표준에 해당하는 의미를 갖고 있다.
```

Comment:
According to Google Translation,
This line translated to
```
The meaning of the annotation clause is implementation dependent.  On the Java platform, the following scalar annotations have a meaning equivalent to the standard.
```

Since table is removed, I think these sentences are no longer necessary